diff --git a/node_modules/awesome-json2json/lib/Json2json.js b/node_modules/awesome-json2json/lib/Json2json.js
index 3038675..9e8efaa 100644
--- a/node_modules/awesome-json2json/lib/Json2json.js
+++ b/node_modules/awesome-json2json/lib/Json2json.js
@@ -27,7 +27,7 @@ var Json2json = /** @class */ (function () {
         var currentJSON = this.getJSONByPath(json, fullTemplate.$path, context);
         if (fullTemplate.$disable) {
             if (this.isArrayTemplate(fullTemplate)) {
-                currentJSON = currentJSON.filter(function (currentJSONItem) {
+                currentJSON = currentJSON && currentJSON.filter(function (currentJSONItem) {
                     return !(fullTemplate.$disable(currentJSONItem, __assign({}, context, { $item: currentJSONItem })));
                 });
             }
@@ -47,7 +47,7 @@ var Json2json = /** @class */ (function () {
         }
         if (fullTemplate.$formatting) {
             if (this.isArrayTemplate(fullTemplate)) {
-                currentJSON = currentJSON.map(function (currentJSONItem) {
+                currentJSON = currentJSON && currentJSON.map(function (currentJSONItem) {
                     return fullTemplate.$formatting(currentJSONItem, __assign({}, context, { $item: currentJSONItem }));
                 });
             }
@@ -67,7 +67,7 @@ var Json2json = /** @class */ (function () {
         var filteredKeys = Object.keys(fullTemplate).filter(function (key) { return !(/^\$/.test(key)); });
         if (this.isArrayTemplate(fullTemplate)) {
             var index_1 = 0;
-            return currentJSON.map(function (currentJSONItem) {
+            return currentJSON &&  currentJSON.map(function (currentJSONItem) {
                 var result = {};
                 filteredKeys.forEach(function (key) {
                     var childResult = _this.mapChild(currentJSONItem, fullTemplate[key], __assign({}, context, { $item: currentJSONItem, $index: index_1 }));
